mayastor/tests/replica_timeout.rs

```diff
@@ -1,156 +1,101 @@
+    for h in &mut hdls {
+        h.bdev.list(Null {}).await.unwrap();
+        h.bdev
+            .create(BdevUri {
+                uri: "malloc:///disk0?size_mb=100".into(),
+            })
             .await
-            .expect_err("should fail read after controller reset");
-        ms = handle.join().unwrap();
-        bdev_io::read_some(NXNAME, 0, 0xff)
+            .unwrap();
+        h.bdev
+            .share(BdevShareRequest {
+                name: "disk0".into(),
+                proto: "nvmf".into(),
+            })
             .await
-            .expect("should read again after Nexus child continued");
-        nexus_lookup(NXNAME).unwrap().destroy().await.unwrap();
-        assert!(nexus_lookup(NXNAME).is_none());
-    });
-
-    common::delete_file(&[DISKNAME1.to_string()]);
-}
-
-#[test]
-#[ignore]
-fn replica_term() {
-    generate_config();
-
-    common::truncate_file(DISKNAME1, DISKSIZE_KB);
-    common::truncate_file(DISKNAME2, DISKSIZE_KB);
-
-    let mut ms1 = start_mayastor(CFGNAME1);
-    let mut ms2 = start_mayastor(CFGNAME2);
-    // Allow Mayastor processes to start listening on NVMf port
-    thread::sleep(time::Duration::from_millis(250));
-
-    test_init!();
+            .unwrap();
+    }
 
-    Reactor::block_on(async {
-        create_nexus(false).await;
-        bdev_io::write_some(NXNAME, 0, 0xff).await.unwrap();
-        bdev_io::read_some(NXNAME, 0, 0xff).await.unwrap();
-    });
-    ms1.sig_term();
-    thread::sleep(time::Duration::from_secs(1));
-    Reactor::block_on(async {
-        bdev_io::read_some(NXNAME, 0, 0xff)
-            .await
-            .expect("should read with 1 Nexus child terminated");
-    });
-    ms2.sig_term();
-    thread::sleep(time::Duration::from_secs(1));
-    Reactor::block_on(async {
-        bdev_io::read_some(NXNAME, 0, 0xff)
+    let mayastor = MayastorTest::new(MayastorCliArgs::default());
+
+    mayastor
+        .spawn(async move {
+            nexus_create(
+                NXNAME,
+                1024 * 1024 * 50,
+                None,
+                &[
+                    format!(
+                        "nvmf://{}:8420/nqn.2019-05.io.openebs:disk0",
+                        hdls[0].endpoint.ip()
+                    ),
+                    format!(
+                        "nvmf://{}:8420/nqn.2019-05.io.openebs:disk0",
+                        hdls[1].endpoint.ip()
+                    ),
+                ],
+            )
             .await
-            .expect_err("should fail read with 2 Nexus children terminated");
+            .unwrap();
+            bdev_io::write_some(NXNAME, 0, 0xff).await.unwrap();
+            bdev_io::read_some(NXNAME, 0, 0xff).await.unwrap();
+        })
+        .await;
+
+    test.pause("ms1").await.unwrap();
+    let mut ticker = tokio::time::interval(Duration::from_secs(1));
+    for i in 1 .. 6 {
+        ticker.tick().await;
+        println!("waiting for the container to be fully suspended... {}/5", i);
```

Comment:
Yeah, actually im not sure -- because of its not a log message from mayastor itself, but a test message. and If we were to use log::info() it should be "recorded" as if it were a log message from mayastor... if that makes sense.

Reply:
hmm not sure, it'd be nice to should show up as something like mayastor::test::test_name